### ISSUE TITLE
[GS v4] Intentional kerning changes in downstream Berlow GS v4 sources

### ISSUE THREAD

**Issue by madig** [googlefonts/googlesans#236]
@sannorozco mentioned in an email that 

> The only intentional changes that I’m aware are the kerning of /period /comma with /quoteright /quotedblright

So I looked at the upstream and mater Berlow sources and found that the aforementioned glyphs are part of the classes `{'public.kern1.period', 'public.kern1.quoteright', 'public.kern2.period', 'public.kern2.quoteright'}` without any bare glyphs-to-anything kerning (good!).

All upstream masters have the kern pair 

1. (('public.kern1.quoteright', 'public.kern2.quoteright'), -30)

All Berlow masters have the kern pairs

1. (('public.kern1.period', 'public.kern2.quoteright'), -80)
2. (('public.kern1.quoteright', 'public.kern2.quoteright'), -30)

So there is just one new one across everything. Did I miss anything? Are there new groups that were intentionally added or changed?

Script:

```python
from fontTools.designspaceLib import DesignSpaceDocument
from ufoLib2 import Font

glyphs = {"period", "comma", "quoteright", "quotedblright"}

up = DesignSpaceDocument.fromfile("source/GoogleSans/GoogleSans.designspace")
it = DesignSpaceDocument.fromfile("source/GoogleSans/GoogleSans-Italic.designspace")
# up = DesignSpaceDocument.fromfile(
#     "../GoogleSans-fb/sources/GS Cubic Sources/Regular/GS Regular Berlow FinalFull.designspace"
# )
# it = DesignSpaceDocument.fromfile(
#     "../GoogleSans-fb/sources/GS Cubic Sources/Italic/GS Italic Berlow FinalFull.designspace"
# )

for ufo in up.loadSourceFonts(Font.open) + it.loadSourceFonts(Font.open):
    groups = {k for k, v in ufo.groups.items() if any(g in glyphs for g in v)}
    kernkeys = glyphs.union(groups)
    kerns = [(k, v) for k, v in ufo.kerning.items() if all(g in kernkeys for g in k)]
    print(ufo.info.styleName, kerns)
```

---

Related #234

**Comment by sannorozco** [comment]
Hi @madig,

The updated kerning was -80 units on all masters, Roman & Italic, just updating quoteright and groups updated the rest

[Kerning update](https://github.com/googlefonts/GoogleSans-fb/commit/db9db3da90ddf9f6a92521934f50c75fa9146a8b)

```
period quoteright
comma quoteright
period quotedblright
comma quotedblright
```

<img width="1676" alt="Screen Shot 2021-03-01 at 10 24 07 AM" src="https://user-images.githubusercontent.com/1184152/109526602-5d9a7580-7a78-11eb-8f0e-eae481250e7d.png">

**Comment by chrissimpkins** [comment]
Moving this to the googlefonts/googlesans repository so that we can track it there after the source move.

**Comment by chrissimpkins** [comment]
Merged into #240

**Comment by chrissimpkins** [comment]
Merged to main
